doc/devices/dts/device_tree.rst

```diff
@@ -173,6 +173,8 @@ The following is a more precise list of required files:
   * Add a board level .dts file that includes the SoC family .dtsi files
     and enables the nodes required for that specific board.
   * Board .dts file should specify the SRAM and FLASH devices, if present.
+    * Flash device node might specify `flash partitions`_.
```

Comment:
How properly (for zephyr-project doc) makes such link to the *.rts file in the same directory.